core/src/main/java/org/mobilitydata/gtfsvalidator/testing/LoadingHelper.java

```diff
@@ -63,6 +63,7 @@ public <X extends GtfsEntity, Y extends GtfsTableContainer<X>> Y load(
       GtfsTableDescriptor<X> tableDescriptor, String... lines) throws ValidatorLoaderException {
     String content = Arrays.stream(lines).collect(Collectors.joining("\n"));
     InputStream in = new ByteArrayInputStream(content.getBytes(StandardCharsets.UTF_8));
+    noticeContainer = new NoticeContainer(); // reset notices if reloading helper
```

Comment:
@bdferris-v2 just FYI, I added this here to clear notices when calling `load` so that I could call it multiple times in one test.  I don't think it it's a problem, and all tests pass, but let me know if you have any concerns.

Reply:
I think that's probably fine.